packages/web-components/@sfx/sayt/src/sayt.ts

```diff
@@ -0,0 +1,88 @@
+import { LitElement, customElement, html, property } from 'lit-element';
+// import { Base } from '@sfx/base';
+import '@sfx/autocomplete';
+import { SAYT_EVENT } from './events';
+
+/**
+ * The `sfx-sayt` component is responsible for displaying and hiding the
+ * `sfx-autocomplete` and `sfx-products` components.
+ */
+@customElement('sfx-sayt')
+// TODO: look into the Base class so this class can extend it again. The
+// issue that we're seeing is related to changing properties and having
+// the DOM update accurately in response to those property changes.
+export default class Sayt extends LitElement {
+  /**
+   * Determines if the `sfx-autocomplete` component will be hidden or not.
+   */
+  @property({ type: Boolean, reflect: true }) hideAutocomplete = false;
+  /**
+   * Determines the visibility of the `sayt` component.
+   */
+  @property({ type: Boolean, reflect: true }) show = false;
+
+  constructor() {
+    super();
+    this.handleVisibilityEvent = this.handleVisibilityEvent.bind(this);
+  }
+
+  /**
+   * Register event listeners.
```

Comment:
Doc comments are written in the third person.

```suggestion
   * Registers event listeners.
```

Reply:
Updated.